README.md

```diff
@@ -1,137 +1,156 @@
-Bounded Counter – Frequency estimation for large sets
-======================================================================
+# Bounter -- Counter for large datasets
 
-**Work in progress!**
+[![Build Status](https://travis-ci.org/RaRe-Technologies/bounter.svg?branch=master)](https://travis-ci.org/RaRe-Technologies/bounter)[![GitHub release](https://img.shields.io/github/release/rare-technologies/bounter.svg?maxAge=3600)](https://github.com/RaRe-Technologies/bounter/releases)[![Mailing List](https://img.shields.io/badge/-Mailing%20List-lightgrey.svg)](https://groups.google.com/forum/#!forum/gensim)[![Gitter](https://img.shields.io/badge/gitter-join%20chat%20%E2%86%92-09a3d5.svg)](https://gitter.im/RaRe-Technologies/gensim)[![Follow](https://img.shields.io/twitter/follow/spacy_io.svg?style=social&label=Follow)](https://twitter.com/gensim_py)
 
-Count frequencies in massive data sets using fixed memory footprint with a smart
-[Count-min Sketch](https://en.wikipedia.org/wiki/Count%E2%80%93min_sketch) and HashTable implementation.
+Bounter is a Python library, written in C, for extremely fast probabilistic counting of item frequencies in massive datasets, using only a small fixed memory footprint.
 
-Contains implementation of Count-min Sketch table with conservative update, as well as our own implementation
-using logarithmic probabilistic counter for reducing the space requirement (*log1024, log8*).
-Uses an embedded [hyperloglog](https://github.com/ascv/HyperLogLog) counter to estimate the cardinality of the set of elements (how many distinct elements are tracked).
+## Why Bounter?
 
-The package also contains a fast HashTable counter allocated with bounded memory which removes its low-count objects
-instead of expanding.
+Bounter lets you count how many times an item appears, similar to Python's built-in `dict` or `Counter`:
 
 ```python
-cms = CountMinSketch(size_mb=512) # Use 512 MB
-print(cms.width)  # 16 777 216
-print(cms.depth)   # 8
-print(cms.size)  # 536 870 912 (512 MB in bytes)
-
-cms.increment("foo")
-cms.increment("bar")
-cms.increment("foo")
-
-print(cms["foo"]) # 2
-print(cms["bar"]) # 1
-print(cms.cardinality()) # 2
-print(cms.total()) # 3
+from bounter import bounter
+
+counts = bounter(size_mb=1024)  # use at most 1 GB of RAM
+counts.update([u'a', 'few', u'words', u'a', u'few', u'times'])  # count item frequencies
+
+print(counts[u'few'])  # query the counts
+2
```

Comment:
Better move this output to comment (as it was before), because user typically copy-paste  these examples to console (same for all code examples)